roc/convert.go

```diff
@@ -37,3 +39,34 @@ func go2cBool(b bool) C.uint {
 	}
 	return 0
 }
+
+func go2cUlonglongs(tds map[string]time.Duration) (map[string]C.ulonglong, error) {
```

Comment:
I think using map looks over-complicated and someone who reads the code will spend time wondering why it's here.

Let's instead create functions for single value:

```go
func go2cUnsignedDuration(d time.Duration) ulonglong
func c2goUnsignedDuration(d ulonglong) time.Duration
```

No need for any wrapper function, just create an alias like we do it for `C.char` at the top of this file:

```go
type ulonglong = C.ulonglong
```

No need to pass field name to those functions; instead, please add field name to the error on the caller side, like we do for go2cStr (find its usages in code)

Reply:
Morgen! Morgen! Nur nicht heute!